contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveAuthorizationHelper.java

```diff
@@ -146,6 +147,12 @@ private void authorize(final HiveOperationType hiveOpType, final List<HivePrivil
     } catch (final HiveAccessControlException e) {
       throw e;
     } catch (final Exception e) {
+      if (toRead.size() == 1 && InfoSchemaConstants.IS_SCHEMA_NAME.equalsIgnoreCase(toRead.get(0).getObjectName())) {
```

Comment:
I think you meant `'hive.%'`, but not `'hi.%'`

Am I right, it solves the following case:
```
USE hive.some_db;
SHOW SCHEMAS LIKE 'hive.%'
```
But why it should be solved in scope of exception handling? What exception is obtained and why it is obtained?
Looks like it is a fix for consequence, but not the root cause of the issue.

Reply:
removed, and created separate issue https://issues.apache.org/jira/browse/DRILL-6923